common-npm-packages/docker-common-v2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-docker-common-v2",
-  "version": "2.0.4",
+  "version": "2.0.5",
```

Comment:
Bump-up the version to `2.0.6` given the other [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/15114)